README.md

```diff
@@ -116,22 +115,28 @@ The valid values for `ABC_LOG_LEVEL` are `debug`, `info`, `notice`, `warning`,
 The golden-test feature is essentially unit testing for templates. You provide
 (1) a set of template input values and (2) the expected output directory
 contents. The test framework verifies that the actual output matches the
-expected output, using the `verify` subcommand. Separately, the `record` command
+expected output, using the `verify` subcommand. Separately, the `record` subcommand
 helps with capturing the current template output and saving it as the "expected"
 output for future test runs. This concept is similar to "snapshot testing" and
-"[rpc replay testing](https://pkg.go.dev/cloud.google.com/go/rpcreplay)."
+"[rpc replay testing](https://pkg.go.dev/cloud.google.com/go/rpcreplay)." 
+In addition, the `new-test` subcommand create a new golden test to initialize the needed 
+golden test directory structure and `test.yaml`.
 
 Each test is configured by placing a file named `test.yaml` in a subdirectory of
 the template named `testdata/golden/<your-test-name>`. See below for details on
 this file.
 
 Usage:
 
-- `abc templates golden-test record [--test-name=<test_name>] <location>`
-- `abc templates golden-test verify [--test-name=<test_name>] <location>`
+- `abc templates golden-test new-test [options] --test-name=<test_name> [<location>]`
+   see `abc-templates golden-test new-test --help` for supported options.
+- `abc templates golden-test record [--test-name=<test_name>] [<location>]`
+- `abc templates golden-test verify [--test-name=<test_name>] [<location>]`
 
 Examples:
 
+- `abc templates golden-test new-test --test-name=basic examples/templates/render/hello_jupiter`
```

Comment:
I think the same problem appears here. There is no `--test-name` flag. And also at line 131.

Reply:
done